security/cmd/node_agent/na/nodeagent.go

```diff
@@ -41,7 +41,7 @@ type cAGrpcClientImpl struct {
 // SendCSR sends CSR to CA through GRPC.
 func (c *cAGrpcClientImpl) SendCSR(req *pb.Request, pc platform.Client, cfg *Config) (*pb.Response, error) {
 	if cfg.IstioCAAddress == "" {
-		return nil, fmt.Errorf("Istio CA address is empty")
+		return nil, fmt.Errorf("istio CA address is empty")
```

Comment:
Istio is used with the first letter capitalized. Can we ignore this?

Reply:
Sure. Thanks for letting me know.